docs/users/author-guide.md

```diff
@@ -1,28 +1,106 @@
+
+ [runBioSimuations](https://run.biosimulations.org/create) provides a simple web form for building COMBINE/OMEX archives with SED-ML files from model files (e.g., CellML, SBML). This tool support all of the modeling languages supported by BioSimulations. 
+
+More detailed information about creating SED-ML files to work with BioSimulations can be found at [here](../concepts/Experiments.md).
 
 ### Visualizations
 
-### Metadata
+BioSimulations recommends [Vega](https://vega.github.io/vega/) for data visualizations of simulation results. Vega is a powerful, declarative grammar for describing interactive, two-dimensional data visualizations.
+
+One key feature of Vega is that it modularly captures the graphical marks which comprise visualizations and how those graphical marks should be painted with data. This feature makes it easy to produce data visualizations for multiple simulation conditions by combining the same graphical marks with results from multiple simulations. This features also makes the provenance of data visualizations transparent. As a result, Vega is ideal for collaboration and publication.
+
+Vega files can be included in the ComBINE/OMEX archive of a simulation project to enable visualization on the [project view page](./user-guide.md#visualizations).
+
+More information on how to create Vega files that can incorporate data from BioSimulations projects can be found at [here](../concepts/Visualizations.md).
+
+### Metadata 
+BioSimulations uses the [OMEX-Meta](https://co.mbine.org/standards/omex-metadata) specification to capture metadata about the simulation project. BioSimulations has a list of recommended metadata fields that are used to display the [metadata](./user-guide.md#metadata) of a simulation project. Information about how to use theses fields can be found [here](../concepts/Metadata.md).
 
 ### COMBINE/OMEX archives
 
+BioSimulations uses COMBINE/OMEX archives to bundle the multiple files typically involved in modeling projects into a single archive. These files include, but are not limited to:
+
+- Models (e.g., in CellML, SBML format)
+- Simulation experiments (SED-ML files)s
+- Visualizations for visualizing simulation results (e.g., in Vega format)
+- Supplementary files, such as data used to calibrate the model
+- Metadata about the simulation project (RDF files that follow the OMEX Metadata guidelines)
+ 
 ## Validating projects
 
+BioSimulations provides a validation tool for validating the contents of a COMBINE/OMEX archive. This tool is available on [runBioSimulations](https://run.biosimulations.org/utils/validate-project). The user can upload a COMBINE/OMEX archive or provide a URL to a COMBINE/OMEX archive. The tool will validate the contents of the archive and report any errors. The interface provides a customizable report of the validation results, including validation of the model file, Sed-ML file, metadata, archive manifest, and any images presented in the archive.
```

Comment:
This report is not customizable by users.

Reply:
"Sed-ML" is spelled "SED-ML"

Reply:
By customizable i was referring to the various options on selecting which parts of the project to validate. I can see how this is confusing. I'll reword this